packages/driver/src/conUtils.ts

```diff
@@ -587,7 +587,7 @@ async function parseConnectDsnAndArgs(
       tlsCA: `'tlsCA' option`,
       tlsCAFile: `'tlsCAFile' option`,
       tlsSecurity: `'tlsSecurity' option`,
-      tlsServerName: `'tlsServerName' option`,
+      tlsServerName: `tlsServerName option`,
```

Comment:
another stray edit 😅

Reply:
it's weird, I kinda remember you already mentioned these fixes in some PR and I've done them...